e2e/impl/tokens.go

```diff
@@ -32,6 +32,46 @@ func (ctx *ValidationContext) sendTokensAndWait(from, to accounts.Account, token
 }
 
 func (ctx *ValidationContext) sendTokens(from, to accounts.Account, tokens int64) error {
+	weis := toWei(tokens)
+	hexWeis := hexutil.Big(*weis)
+	args := knode.TransferArgs{
+			From:  from.Address,
+			To:    &to.Address,
+			Value: &hexWeis,
+	}
+
+	res := &cluster.ExecResponse{}
+	return ctx.execCommand(transferTokens(args), res)
+}
+
+func (ctx *ValidationContext) mintTokensAndWait(governance []accounts.Account, to accounts.Account, tokens int64) error {
+	res := &cluster.ExecResponse{}
+	if err := ctx.execCommand(getTokenBalance(to.Address), res); err != nil {
```

Comment:
we could add this as a function in the parent context like `balance, err := ctx.getTokenBalance(to.Address)`, seems useful